CONTRIBUTING.md

```diff
@@ -88,6 +88,36 @@ bin)/gulp` instead of `gulp`.)
 Note that running the tests also requires an installation of
 Chrome v59 or higher (v60 if you're on Windows).
 
+#### Visual regression testing
+
+The Standards come with optional tooling for detecting visual regressions,
+which can be especially useful if you're refactoring CSS.
+
+These tests work by comparing current screenshots of the Standards' Fractal
+components to "golden" screenshots that represent what the components are
+supposed to look like.
+
+Golden screenshots are stored on your local development system *only*;
+they're not version controlled.
+
+To generate the golden screenshots, run:
+
+```
+node spec/visual-regression-tester.js --updateGolden
+```
+
+Then, make any CSS refactorings (or switch to a branch that has them).
+
+To compare the current state of your CSS to the golden screenshots, run:
+
+```
+node spec/visual-regression-tester.js
```

Comment:
Hmm, my only concerns about that are that folks are likely going to want to run this with arguments. They can run the tester with `--help` to get CLI help, and as with mocha, they can run it with `-g <pattern>` to only run tests that match a certain pattern, which is particularly useful because these tests take longer to run than pure unit tests.

While passing extra arguments to npm/yarn *can* be done with  `--`, it adds extra typing and isn't very discoverable. With npm, updating the golden screenshots would become `npm run visual-regression-tester -- --updateGolden` which is more of a mouthful. Also, unless we come up with a shorter name, `visual-regression-tester` takes long to type and can't be easily autocompleted with <kbd>Tab</kbd> like `spec/visual-regression-tester.js` can.

Another annoying thing with npm < 5 (but fixed in yarn and npm 5) is the huge amount of debug output that appears when something fails, which this command is frequently expected to.

So, I guess I just don't feel like this is a very good fit as an `npm` script, but if you feel strongly about it, we can convert it over.